installers/sirepo-dev/radiasoft-download.sh

```diff
@@ -40,4 +40,13 @@ sirepo_dev_main() {
     cd sirepo
     npm install
     cd ..
+    sirepo_dev_npm
+}
+
+sirepo_dev_npm() {
+    if ! [[ $(type -p karma) && $(type -p jshint) ]]; then
+       cd ~/src/radiasoft/sirepo
```

Comment:
Can be removed.

Reply:
09ed5adf527e153ce4b1c312f35d0f1542891da6